sw/device/tests/plic_all_irqs_test_helper.h

```diff
@@ -0,0 +1,83 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_PLIC_ALL_IRQS_TEST_HELPER_H_
+#define OPENTITAN_SW_DEVICE_TESTS_PLIC_ALL_IRQS_TEST_HELPER_H_
```

Comment:
maybe this should be in sw/device/lib/testing/rv_plic_all_irqs_testutils.h ?

Reply:
If I did, it would imply that other tests can use these macros, which is not the case. Perhaps leave it here?